docker/base-images/Makefile

```diff
@@ -20,17 +20,17 @@ base-ci-builder:
 	docker push temporalio/base-ci-builder:$(DOCKER_IMAGE_TAG)
 
 ##### docker buildx #####
-buildx-docker-container:
+docker-buildx-container:
 	docker buildx create --driver docker-container --use
 
 base-admin-tools-x:
```

Comment:
Why do these auto-push (`type=registry`) if the server ones don't? If you want to build locally, you can just build base images and then server images all locally, right? Pushing should be explicit, I think (maybe one make target to push everything?)

Reply:
Added variable.